examples/README.md

```diff
@@ -70,13 +70,13 @@ When the services started with Docker Compose are ready, you will need to start
 examples in order:
 
 ```shell
-go run examples/seventh/main.go
-go run examples/first/main.go
-go run examples/second/main.go
-go run examples/third/main.go
-go run examples/fourth/main.go
-go run examples/fifth/main.go
-go run examples/sixth/main.go examples/sixth/greeter.pb.go
+go run examples/http-cache/main.go
+go run examples/http/main.go
+go run examples/kafka/main.go
+go run examples/amqp/main.go
+go run examples/sns/main.go
+go run examples/sqs/main.go
+go run examples/grpc/main.go examples/grpc/greeter.pb.go
```

Comment:
(Sorry in advance for the nitpicky comment)

There's actually no `examples/grpc/greeter.pb.go` path. There is a `./component/grpc/greeter/greeter.pb.go`, but I'm not sure if it's required to run the examples.

Reply:
> I felt like it wasn't something to do as part of this PR and intentionally left it out.
Okay, I see your point. I'll try to run the examples and see if we need another issue.

Reply:
I also had issues with the SNS and SQS examples (it wouldn't create the queues) and had a conflict with two examples trying to run on the same port.

Reply:
Thanks for the feedback! The SNS/SQS examples should be okay now at `master (fixed by #282); and the ports conflict is being tracked by #277. The examples should be a-okay after the latter is resolved!